docs/compass/02-03-runtime-agent-workflow.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Runtime Agent architecture
+type: Architecture
+---
+
+This document presents the workflow of Runtime Agent. 
+
+![Runtime Agent architecture](./assets/runtime-agent-architecture.svg)
```

Comment:
Please use grey for the shape outlines, as suggested in the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/02-diagrams.md#outlines). Moreover, the connectors seem to have too many curves that can easily be straightened.